### DISCUSSION TITLE
Duda Trabajo Practico Complejidad y Metodo Maestro

### DISCUSSION THREAD

**Discussion by EmilianoGermani** [harpomaxx/algoritmos2#56] [General]
Tengo una duda respecto al ejercicio 4 del trabajo practico de complejidad, ya que en caso de que el numero que se encuentra en mitad de la lista ya es el menor o mayor de toda la lista, en dicho caso no podría tener la mitad de valores mayores y menores a su lado izquierdo, ¿Cómo debe evaluarse a la hora de buscar un valor para dejarlo en la mitad de la lista y así cumplir con lo pedido en la consigna, o se busca un valor de forma aleatoria (como el pivote en Quicksort)?
![Sin título](https://github.com/harpomaxx/algoritmos2/assets/105568147/16355a75-d4b8-4c39-afb5-200c7f39fad5)

**Comment by harpomaxx** [comment]
La idea del ejercicio es dada una lista con valores, ordernarlo de una manera en particular. La manera para orderlo es la siguiente:
Hay que colocar un numero en la metidad de la lista, para que luego cada mitad contenga la mitdad de los elementos menores que el.

Justamente es tu tarea encontrar ese numero y colocarlo a la mitad.  No es que se encuentra en la mitad, sino que vos debes encargarte de dejar la lista ordenada de la manera que se especifica el ejercicio. 

BTW, perdon por la demora es responder.
